src/content/developers/docs/scaling/zk-rollups/index.md

```diff
@@ -1,34 +1,206 @@
 ---
 title: Zero-Knowledge Rollups
-description: Introduction to zero-knowledge rollups
+description: An introduction to zero-knowledge rollups, a scaling solution used by the Ethereum community. 
 lang: en
 sidebar: true
----
+--- 
+
+Zero-knowledge rollups (ZK-rollups) are layer 2 [scaling solutions](/developers/docs/scaling/) that improve throughput on Ethereum Mainnet through off-chain execution. ZK-rollups process transactions independently, but post state data to Mainnet as `calldata`. They also produce cryptographic proofs of validity for transaction batches to guarantee the correctness of off-chain computation. 
 
 ## Prerequisites {#prerequisites}
 
 You should have a good understanding of all the foundational topics and a high-level understanding of [Ethereum scaling](/developers/docs/scaling/). Implementing scaling solutions such as rollups is an advanced topic as the technology is less battle-tested and continues to be researched and developed.
 
 Looking for a more beginner-friendly resource? See our [introduction to layer 2](/layer-2/).
 
-## Zero-knowledge rollups {#zk-rollups}
+## What are zero-knowledge rollups? {#what-are-zk-rollups}
+
+**Zero-knowledge rollups (ZK-rollups)** execute transactions on layer 2 (L2), which they bundle (or "roll-up") into batches. ZK-rollup operators produce [validity proofs](/glossary/#validity-proof) to prove the validity of off-chain transactions before submitting batches on-chain. These proofs can come in the form of ZK-SNARKs (Zero-Knowledge Succinct Non-Interactive Argument of Knowledge) or ZK-STARKs (Zero-Knowledge Scalable Transparent Argument of Knowledge). 
+
+The ZK-rollup smart contract maintains the state of all transactions on the L2 chain, and this state can only be updated with a validity proof. This means that ZK-rollups only need to provide validity proofs to finalize transactions on the layer 1 (L1) chain instead of posting all the transaction data like [optimistic rollups](/developers/docs/scaling/optimistic-rollups/). 
+
+There are no delays when moving funds from ZK-rollups (L2) to Ethereum (L1) because exit transactions are finalized once the ZK-rollup contract verifies the validity proof. Conversely, withdrawing funds from optimistic rollups is subject to a delay to allow anyone to challenge the exit transaction with a [fraud proof](/glossary/#fraud-proof). 
```

Comment:
"Finalized" is a loaded term because it is often used in the context of the consensus mechanism to reference a benchmark degree of economic security for a specific block. At the transaction level, perhaps it is better to say it has been "executed" rather than "finalized" to avoid any confusion.
```suggestion
There are no delays when moving funds from ZK-rollups (L2) to Ethereum (L1) because exit transactions are executed once the ZK-rollup contract verifies the validity proof. Conversely, withdrawing funds from optimistic rollups is subject to a delay to allow anyone to challenge the exit transaction with a [fraud proof](/glossary/#fraud-proof). 
```